testing/tests/DevExpress.ui.widgets/optionChanged_bundled.tests.js

```diff
@@ -38,7 +38,8 @@ define(function(require) {
                     name === 'validationMessageOffset' ||
                     name === 'templatesRenderAsynchronously' ||
                     name === 'ignoreChildEvents' ||
-                    name === '_checkParentVisibility') {
+                    name === '_checkParentVisibility' ||
+                    name === 'showValidationMark') {
```

Comment:
Remove this line and add a code snippet specified below to the "optionChanged" handler:
```js
case 'showValidationMark':
    break;
```

Reply:
OK